app/models/node_group_edge.rb

```diff
@@ -1,6 +1,4 @@
 class NodeGroupEdge < ApplicationRecord
-  validates_presence_of :to_id, :from_id
```

Comment:
does that mean `:to_id` and `:from_id` are not set in the nested nodegroups case?

Reply:
Sorry for the delay. The `:to_id` and `:from_id` fields get set correctly in the `node_group_edges` table when creating nested groups.